pkgs/servers/wsdd/default.nix

```
@@ -19,10 +21,15 @@ stdenv.mkDerivation rec {
     # Increase timeout to socket urlopen
     # See https://github.com/christgau/wsdd/issues/80#issuecomment-76848906
     ./increase_timeout.patch
+    # Implicitly use currently running asyncio loop
+    # The patch is based on this commit:
+    # https://github.com/christgau/wsdd/commit/0aef2d73c01162d746414c25f1556ae510289cf4
+    ./implicitly_yuse_currently_running_asyncio_loop.patch
```

Typo in filename, the actual patch file name would also need to be changed.
```suggestion
    ./implicitly_use_currently_running_asyncio_loop.patch
```

---

Fixed!